--- a/contrib/babelfishpg_tds/src/include/tds_secure.h.orig   2025-03-25 06:32:52.000000000 +0000
+++ b/contrib/babelfishpg_tds/src/include/tds_secure.h        2025-12-02 18:00:09.492544069 +0000
@@ -31,13 +31,6 @@
 #include "libpq/libpq.h"
 #include "port/pg_bswap.h"
 
-#ifdef USE_SSL
-#ifndef HAVE_BIO_GET_DATA
-#define BIO_get_data(bio) (bio->ptr)
-#define BIO_set_data(bio, data) (bio->ptr = data)
-#endif
-#endif
-
 BIO_METHOD *TdsBioSecureSocket(BIO_METHOD * my_bio_methods);
 
 extern int     tds_ssl_min_protocol_version;
--- a/contrib/babelfishpg_tds/src/backend/tds/tds-secure-openssl.c.orig   2025-03-25 06:32:52.000000000 +0000
+++ b/contrib/babelfishpg_tds/src/backend/tds/tds-secure-openssl.c        2025-12-02 18:00:09.492544069 +0000
@@ -855,11 +855,6 @@
  * to retry; do we need to adopt their logic for that?
  */
 
-#ifndef HAVE_BIO_GET_DATA
-#define BIO_get_data(bio) (bio->ptr)
-#define BIO_set_data(bio, data) (bio->ptr = data)
-#endif
-
 static int
 my_sock_read(BIO * h, char *buf, int size)
 {
